buildsys.mk.in

```diff
@@ -953,4 +952,6 @@ DELETE_OK = printf "@TERM_EL@@TERM_SETAF4@Deleted @TERM_BOLD@%s@TERM_SGR0@@TERM_
 DELETE_FAILED = err=$$?; printf "@TERM_EL@@TERM_SETAF1@Failed to delete @TERM_BOLD@%s@TERM_SGR0@@TERM_SETAF1@!@TERM_SGR0@\n" "$$i"; exit $$err
 
 .CURDIR ?= .
--include ${.CURDIR}/.deps
+include ${.CURDIR}/.deps
+
+$(V).SILENT:
```

Comment:
Looks like I made a comment on an older version of this file so it's not showing up.  To recap, this produces way too much extraneous output.  I'm not merging it.